extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -27,40 +33,37 @@ public function testLoadDataFromRev( $params ) {
 	}
 
 	/**
-	 * @param Revision $rev
 	 * @param array $params
-	 *
-	 * @return PHPUnit_Framework_MockObject_MockObject|WallNotificationEntity
 	 */
-	private function getEntityMock( Revision $rev, array $params ) {
-		$wm = $this->getWallMessageMock( $params );
-
-		$entity = $this->getMockBuilder( 'WallNotificationEntity' )
-			->setMethods( [
-				'getUserName',
-				'getWallMessageFromRev',
-			] )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$entity->expects( $this->once() )
-			->method( 'getUserName' )
-			->willReturn( $params['entity']['userName'] );
-		$entity->expects( $this->once() )
-			->method( 'getWallMessageFromRev' )
-			->with( $this->equalTo( $rev ) )
-			->willReturn( $wm );
-
-		return $entity;
+	private function setupEnvironmentMocks( array $params ) {
+		$userMock = $this->getMock( User::class, [ 'getName' ] );
+		$userMock->expects( $this->once() )
+			->method( 'getName' )
+			->willReturn( $params['user']['userName'] );
+
+		$cacheMock = $this->getMockForAbstractClass( BagOStuff::class );
+		$cacheMock->expects( $this->once() )
+			->method( 'set' )
+			->with( WallNotificationEntity::getMemcKey( $params['id'] ), $params['data'] );
+
+		$messageMock = $this->getMock( Message::class, [ 'escaped' ] );
+		$messageMock->expects( $this->any() )
+			->method( 'escaped' )
+			->willReturn( static::A_FANDOM_USER );
+
+		$this->mockClass( User::class, $userMock, 'newFromID' );
+		$this->mockGlobalVariable( 'wgMemc', $cacheMock );
+		$this->mockGlobalVariable( 'wgCityId', $params['wgCityId'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgCityId" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)